### ISSUE TITLE
Display R-generated figures in R markdown file

### ISSUE THREAD

**Issue by Bai-Li-NOAA** [Andrea-Havron-NOAA/FIMS-software#11]
To display your R figures, you could try to include fig.width or fig.height arguments in the code below. For example `{r, eval = FALSE, fig.height = 4}`

https://github.com/Andrea-Havron-NOAA/FIMS-software/blob/e9f19474c3de9dcfdfb40a6fcf36cc4b8cf3e72b/docs/ar1xar1.Rmd#L64

**Comment by Andrea-Havron-NOAA** [comment]
Thanks @Bai-Li-NOAA! The issue is that I do not want to run the r code to generate the plots - model validation especially can take a long time. I've saved the images and would like to embed them directly into the html, however, it is a bit more complicated to embed images from github. I have tried without success:
```{r, echo = FALSE}
url <- "https://github.com/Andrea-Havron-NOAA/FIMS-software/tree/master/src/Rcpp/ar1xar1/correct.png"
```
![](url) 

and 

<image, src = "`r url`">
